apps/docs/src/pages/index.mdx

```diff
@@ -14,21 +14,21 @@ Assuming that a question has been solved in your server and no where else on the
 | 2. Find the question indexed from your server in Google          | 2. No relevant results found                                                                                               |
 | 3. Read the answer to the question                               | 3. Try to find a place to ask the question                                                                                 |
 | 4. Join the server linked on the question for further discussion | 4. Find a Discord server and join it (if the invite link is still valid)                                                   |
-|                                                                  | 5. Search in the server I just joined for the question using the small Discord search sidebar                              |
-|                                                                  | 6. (Maybe) Find the question I'm looking for                                                                               |
-|                                                                  | 7. If I don't find the question or don't search for it, I ask my question in the server, resulting in a duplicate question |
```

Comment:
I sort of want to keep this in here, or find different phrasing on it since it highlights a pain point of repeat questions